code/game/machinery/hologram.dm

```diff
@@ -90,6 +97,14 @@ var/const/HOLOPAD_MODE = RANGE_BASED
 				return
 			if(last_request + 200 < world.time) //don't spam other people with requests either, you jerk!
 				last_request = world.time
+				var/obj/item/card/id/I = user.GetIdCard()
+				if(!user.GetIdCard())
+					to_chat(user, span("notice", "You need authorization to use the holocall system. Please equip a valid ID card."))

```

Comment:
If you want to require a ID card you need to `return` or the execution won't be aborted.

Reply:
return is in the next line already

Reply:
there already is a return on the next line